lib/webengine.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
 **
-** Copyright (C) 2016 Jolla Ltd.
+** Copyright (C) 2020 Jolla Ltd.
```

Comment:
I wouldn't mind if you change capital C to lower case in this line but keep the 2016 and add:

```
Copyright (c) 2020 Open Mobile Platform LLC.
```